core/src/main/scala/org/http4s/headers/Strict-Transport-Security.scala

```diff
@@ -56,7 +57,33 @@ object `Strict-Transport-Security`
     fromLong(maxAge, includeSubDomains, preload).fold(throw _, identity)
 
   def parse(s: String): ParseResult[`Strict-Transport-Security`] =
-    HttpHeaderParser.STRICT_TRANSPORT_SECURITY(s)
+    ParseResult.fromParser(parser, "Invalid `Strict-Transport-Security` header")(s)
+
+  private[http4s] val parser: Parser[`Strict-Transport-Security`] = {
+    val maxAge: Parser1[`Strict-Transport-Security`] =
+      (Parser.string1("max-age=") *> AdditionalRules.Digits).map { (age: Long) =>
```

Comment:
Is this case sensitive?  I bet it's not, but I'd have to find the RFC.

Reply:
https://tools.ietf.org/html/rfc6797#section-6.1

> Directive names are case-insensitive.

thanks you are right, will change